src/handlers/pricing/action.ts

```diff
@@ -11,6 +12,12 @@ export const pricingLabelLogic = async (): Promise<void> => {
   if (!payload.issue) return;
   const labels = payload.issue.labels;
 
+  let valid = await handleLabelsAccess();
+
+  if (!valid) {
+    return;
```

Comment:
we might need to leave the log with a proper message here.

Reply:
> we might need to leave the log with a proper message here.

Its already logged in the function